src/foundation.cs

```diff
@@ -3586,6 +3586,18 @@ interface NSDate : NSSecureCoding, NSCopying, CKRecordValue {
 		[Export ("timeIntervalSinceReferenceDate")]
 		double SecondsSinceReferenceDate { get; }
 
+		[Export ("timeIntervalSinceDate:")]
+		double TimeIntervalSince (NSDate anotherDate);
```

Comment:
it's a method so [it should have a verb](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-methods), e.g. `GetTimeIntervalSince (...)`